core/feed/serializer/opds.py

```diff
@@ -381,7 +381,7 @@ def _serialize_data_entry(self, entry: DataEntry) -> etree._Element:
 
     @classmethod
     def to_string(cls, element: etree._Element) -> bytes:
```

Comment:
It seems really weird here to have a method called `to_string` returning `bytes`. It seems like we should probably either return a string, or rename the method to `to_bytes`.